cmake/modules/hunter_dump_cmake_flags.cmake

```diff
@@ -20,6 +21,24 @@ function(hunter_dump_cmake_flags)
     hunter_internal_error("Unparsed arguments: ${x_UNPARSED_ARGUMENTS}")
   endif()
 
+
+  if(IOS)
+    hunter_test_string_not_empty("${IOS_SDK_VERSION}")
+    string(COMPARE EQUAL "${IOS_DEPLOYMENT_SDK_VERSION}" "" _no_deployment_sdk_version)
+    if(_no_deployment_sdk_version)
+      set(CMAKE_CXX_FLAGS "-miphoneos-version-min=${IOS_SDK_VERSION}")
+      set(CMAKE_C_FLAGS "-miphoneos-version-min=${IOS_SDK_VERSION}")
+    else()
+      set(CMAKE_CXX_FLAGS "-miphoneos-version-min=${IOS_DEPLOYMENT_SDK_VERSION}")
+      set(CMAKE_C_FLAGS "-miphoneos-version-min=${IOS_DEPLOYMENT_SDK_VERSION}")
+    endif()
+  
+    if(XCODE AND CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE)
```

Comment:
I'm not aware of the iOS builds without using Xcode. You can add assertion if you want to be sure:
```cmake
if(IOS)
  # ...
  if(NOT XCODE)
    hunter_internal_error("iOS without Xcode")
  endif()
  # ...
endif()
```